tests/testthat/test-scheduler.R

```diff
@@ -72,26 +72,19 @@ test_that("linear graph, reversed", {
 
 test_that("linear graph, delayed", {
   x <- 1
-  delayed_future <- future.callr::callr({
-    Sys.sleep(1)
-    list(success = TRUE)
-  })
+  delayed_future <- future.callr::callr({ Sys.sleep(1); list(success = TRUE) })
```

Comment:
Curly braces on a single line produce lints. It would be nice to either split this into multiple lines or exempt them in `.lintr`.